src/content/overview/observability/configuration/_index.md

```diff
@@ -0,0 +1,170 @@
+
+Multi-tenancy configuration affects all other platform components, making it essential to understand before configuring other areas.
+
+### Organization management
+
+Manage your observability platform's organizational structure through **[`GrafanaOrganization`]({{< relref "/reference/platform-api/crd/grafanaorganizations.observability.giantswarm.io" >}})** resources. Organization management enables you to:
+
+- **Create isolated workspaces**: Set up dedicated Grafana organizations for teams, environments, or projects
+- **Configure access controls**: Map identity provider groups to appropriate roles (admin, editor, viewer)
+- **Define tenant access**: Control which data tenants each organization can access
+- **Provision datasources**: Automatically configure tenant-scoped access to metrics, logs, and alerting
+
+Effective organization management provides teams with secure, isolated access to their observability data while maintaining proper governance and data separation. Contact your Giant Swarm account engineer for guidance on creating and configuring Grafana organizations with practical examples and step-by-step instructions.
+
+## Data ingestion configuration
+
+Control what observability data flows into your platform and how it's processed:
+
+### Metrics collection
+
+Configure **[metrics ingestion]({{< relref "/tutorials/observability/data-ingestion/metrics" >}})** to collect custom application and infrastructure metrics:
+
+- **ServiceMonitors**: Define which services to scrape for metrics and how frequently
+- **PodMonitors**: Configure direct pod-level metric collection for specialized use cases
+- **Tenant routing**: Specify which tenant receives metrics through labeling
+- **Metric transformation**: Apply relabeling rules to modify, filter, or enrich metrics before storage
+
+### Log collection
+
+Configure **[log ingestion]({{< relref "/tutorials/observability/data-ingestion/logs" >}})** to capture application and system logs:
+
+- **PodLogs**: Select which pods to collect logs from using label selectors
+- **Pod labels**: Enable automatic log collection by labeling pods with tenant information
+- **Log parsing**: Configure structured parsing and label extraction from log content
+- **Namespace filtering**: Control log collection scope by namespace
+
+### External data sources
+
+Integrate data from sources outside Giant Swarm managed clusters through the **[Observability Platform API]({{< relref "/overview/observability/observability-platform-api" >}})**:
+
+- **OIDC authentication**: Configure your identity provider for secure API access
+- **Data ingestion endpoints**: Send metrics, logs, and traces from external systems
+- **OpenTelemetry Protocol (OTLP)**: Configure standardized telemetry data ingestion
+
+## Visualization and dashboard configuration
+
+Customize how you view and visualize your observability data:
+
+### Grafana organizations
```

Comment:
I think this section here is a duplication of the Multi-Tenancy then, right? Doesn't really add any additional value, I would just remote it here. wdyt?